### DISCUSSION TITLE
Compressible Navier Stokes CGFE Resources

### DISCUSSION THREAD

**Discussion by maxnezdyur** [idaholab/moose#19045] [Q&A Modules: Navier-Stokes] [answered]
For the Compressible Navier Stokes CGFE, are there any resources or test input files for it? I found bump.i in the tests section. 
Are there any input files using the Compressible Navier Stokes CGFE method without using the  Modules/Compressible Navier-Stokes System.

**Comment by GiudGiud** [comment, marked as answer]
Hello

There isnt much resource for this apart from the bump test and the step test. You can ask for access to Pronghorn to get more input files using CNS, here: https://ncrcaims.inl.gov/Identity/Account/Login
You can however look at src/actions/CNSAction.C to see what is done by the `Modules/CompressibleNavierStokes` syntax precisely.

Our main focus is finite volume for compressible flows, these two PRs enable it:
https://github.com/idaholab/moose/pull/18395
https://github.com/idaholab/moose/pull/18806

Guillaume

**Reply by maxnezdyur** [reply]
Thank you for the information. May I ask why the focus is on finite volume for compressible flow?

**Reply by GiudGiud** [reply]
Works a lot better. Conservation of quantities is natural, no need for stabilization, no awkwardness at corners for boundary conditions.
